### DISCUSSION TITLE
what is the best practices for deploying etcd

### DISCUSSION THREAD

**Discussion by zgLiuBing** [apache/apisix#5293] [Q&A]
### Issue description

etcd在虚拟机上怎么实现三副本的部署

### Environment

etcd在虚拟机上怎么实现三副本的部署

**Comment by xxm404** [comment]
https://etcd.io/docs/v2.3/docker_guide/

**Comment by shuaijinchao** [comment]
refer to: https://etcd.io/docs/v3.5/demo/
